### DISCUSSION TITLE
Input height/rows

### DISCUSSION THREAD

**Discussion by macdonmc** [twilio-labs/paste#2367] [Q&A] [answered]
Hi there! I'm looking to use a text input that starts as one line and expands upwards as more lines of text are added for a chat feature (similar to how Slack, iMessage, etc work). The project I'm working on is supposed to be entirely dependent on Twilio paste, so I was wondering if this component might be in a backlog to be made or if it already exists and I have missed it.

Thanks for your help!

**Comment by shleewhite** [comment]
Hello @macdonmc! 

We are hoping to release a ChatComposer component in Q2 as a part of our Conversations API UI kit work. Development on these are current paused due to the HAP, but we will follow up in this thread when the component is completed. You can check out [our BPM plan](https://docs.google.com/document/d/1R9J9mHMxCDC4fIMmS0ogJulSHQXJ11AGYzdesPTl5_8/edit#) for more information.

**Comment by loreina** [comment]
👋 Hey @macdonmc,

Just wanted to share an update here. Chat Composer is expected to be part of our upcoming Nov 29, 2022 release. You can follow this [ticket](https://issues.corp.twilio.com/browse/DSYS-3280) to track its progress.

**Reply by macdonmc** [reply]
Awesome to hear! Thank you for keeping me in the loop

**Comment by gloriliale** [comment, marked as answer]
Hi there @macdonmc! Just wanted to let you know that Chat Composer has been released in the newest version of Paste, [see it docs here](https://paste.twilio.design/components/chat-composer)!
